docs/metatype.dev/docs/concepts/features-overview/index.mdx

```diff
@@ -59,9 +59,17 @@ We're taking any and all kinds of feature requests, suggestions and recommendati
       },
       {
         title: "gRPC API",
-        status: "future",
+        status: "complete",
         link: "/docs/reference/typegate",
-        body: <>Support to expose the typegraph through a gRPC API.</>,
+        body: (
+          <>
+             Expose your typegraph through a gRPC API, enabling high-performance,
```

Comment:
> Expose your typegraph through a gRPC API
I think that is not accurate here, _"expose your gRPC API (...)"_ would be the one.